packages/core/src/Stores/notification-store.js

```diff
@@ -1574,4 +1577,27 @@ export default class NotificationStore extends BaseStore {
             this.p2p_completed_orders = response?.p2p_order_list?.list || [];
         }
     }
+
+    showSuccessWalletsUpgradeNotification = () => {
```

Comment:
@aizad-deriv Would you please update the `TNotificationStore` type in `packages/stores/types.ts` 🙏🏻

Reply:
Already done 😃👍